pages/web-development.mdx

```diff
@@ -5,6 +5,20 @@ import { Card, Cards } from "nextra-theme-docs";
 Let's get you setup and ready to build websites and web applications on your
 Laptop or PC.
 
+The minimum techinal specifications for a student computer:
+
+- Processor: Intel Core i5 (or equivalent)
+- 16 GB RAM
+- Hard disk space: at least 20 GB
```

Comment:
Could this be updated to storage rather than Hard disk (or maybe include HDD/SDD or something?)